debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/stackwalker/JITStackWalker.java

```diff
@@ -82,6 +82,7 @@
 import static com.ibm.j9ddr.vm29.structure.J9JavaAccessFlags.*;
 import static com.ibm.j9ddr.vm29.structure.J9StackWalkConstants.*;
 import static com.ibm.j9ddr.vm29.structure.J9StackWalkState.*;
+import static com.ibm.j9ddr.vm29.structure.MethodMetaDataConstants.INTERNAL_PTR_REG_MASK;
```

Comment:
Its not obvious why this change is required, the constant doesn't appear to be used in this file.

Reply:
I must have made an error searching, I can see the use of it now.

Reply:
See line 1222.